src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    reversedRobot[value] = key;
+  }
+
+  if (Object.values(reversedRobot).length < Object.values(robot).length) {
```

Comment:
it works, but let's improve it a bit

it's better to check `If any of the object values are repeated` inside the loop
It will look clearer